udark/udapp/src/main/java/io/underdark/app/model/Node.java

```diff
@@ -151,12 +156,12 @@ public void transportLinkConnected(Transport transport, Link link)
 		routingTable.put(link.getNodeId(),new RoutingInfo(link.getNodeId(),1));
 		activity.refreshPeers();
 		activity.refreshButtons();
-		String rt = encodeRoutingTable();
+		//String rt = encodeRoutingTable();
```

Comment:
Why are we now not encoding and sending out the routing table when we connect with a link? Without this, no one will ever get a message w/ protocol 0